### DISCUSSION TITLE
Cannot publish using traefik

### DISCUSSION THREAD

**Discussion by canytam-krystal** [open-webui/open-webui#5363] [General]
I want to deploy the application behind reverse proxy server TRAEFIK and under a PathPrefix but nothing can shown on browser. Any one know how to do the correct setup?

**Comment by flomickl** [comment]
It is working on my side.
This question needs additional information.
1. Is the container running correctly without traefik and how (run command)
2. what traefik version are you using?
3. traefik run command with container?
4. how does your static and dynamik (or labels) look like to get webui running)?
5. What did you try and do you get any errors?

**Reply by canytam-krystal** [reply]
1. The container running correctly without traefik, I can use browser to connect to the published port.
2. I use traefik v3.1
3. I am not sure about your question. The traefik is worked correctly, I can access other servers via traefik.
4. traefik.yml :
log:
  level: DEBUG

api:
  dashboard: true
  insecure: true

entryPoints:
  web:
    address: ":80"

providers:
  docker:
    endpoint: "unix:///var/run/docker.sock"
    exposedbydefault: false

docker-compose.yml :
labels:
      - "traefik.enable=true"
      - "traefik.http.routers.open-webui.rule=PathPrefix(`/open-webui`)"
      - "traefik.http.routers.open-webui.service=open-webui"
      - "traefik.http.services.open-webui.loadbalancer.server.port=8080"
7. I also try to use middlewares to stripPrefix.
